flink-runtime/src/test/java/org/apache/flink/runtime/registration/RetryingRegistrationConfigurationTest.java

```diff
@@ -21,20 +21,18 @@
 import org.apache.flink.configuration.ClusterOptions;
 import org.apache.flink.configuration.Configuration;
 import org.apache.flink.configuration.TaskManagerOptions;
-import org.apache.flink.util.TestLogger;
 
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 
 import java.time.Duration;
 
-import static org.hamcrest.Matchers.is;
-import static org.junit.Assert.assertThat;
+import static org.assertj.core.api.Assertions.assertThat;
 
 /** Tests for the {@link RetryingRegistrationConfiguration}. */
-public class RetryingRegistrationConfigurationTest extends TestLogger {
+class RetryingRegistrationConfigurationTest {
```

Comment:
Same as above

Reply:
Hi, we reached agreement with Fan Rui and Matthias Pohl previously that we should not add @ExtendWith(TestLoggerExtension.class) explicitly for each class but rather enable it globally in Junit5 configs. See https://issues.apache.org/jira/browse/FLINK-32866